src/main/java/lotto/LottoStatistics.java

```diff
@@ -0,0 +1,37 @@
+package lotto;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class LottoStatistics {
+
+    private final Map<Integer, Integer> matchCounts;
```

Comment:
key의 타입을 Integer 대신 LottoPrize 로 변경하면 어떨까요?
```suggestion
    private final Map< LottoPrize, Integer> matchCounts;
```